base/db/tests/Rcheck_reference.log

```diff
@@ -85,15 +84,13 @@ Namespaces in Imports field not imported from:
 * checking replacement functions ... OK
 * checking foreign function calls ... OK
 * checking R code for possible problems ... NOTE
-db.exists: no visible binding for '<<-' assignment to ‘user.permission’
-db.exists: no visible binding for global variable ‘user.permission’
-load_data_single_run: no visible binding for global variable ‘var_name’
-load_data_single_run: no visible binding for global variable ‘vals’
+convert.input: no visible binding for global variable ‘settings’
+convert.input : log_format_df: no visible binding for global variable
+  ‘.’
```

Comment:
Let's fix these before merging this rather than write them into the log.

Reply:
@infotroph I have removed `settings` variable from `convert.input` but I am not sure how to remove `.` variable. Can you help me out here

Reply:
@infotroph I have removed `.` and `settings` variable from `convert.input`. I am not sure if it's the correct method to do it.